src/screens/SignedChatScreen/index.tsx

```diff
@@ -28,8 +28,8 @@ const SignedChatScreen = () => {
     goToChatInfoScreen,
     sendChat,
     updateChatContinuity,
-    flatListRef: scrollRef,
-    scrollContext
+    scrollContext,
+    flatListRef: scrollRef
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of destructuring has been changed and `updateChatContinuity` has been removed. If `updateChatContinuity` is not used anywhere else in the component, this change is fine. However, if it's used, this will cause a reference error.
```diff
-    updateChatContinuity,
-    flatListRef: scrollRef,
-    scrollContext
+    scrollContext,
+    flatListRef: scrollRef
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->